views/layout.hbs

```diff
@@ -1,6 +1,30 @@
 <!DOCTYPE html>
 <html>
   <head>
+
```

Comment:
the nav bar needs to be the first thing inside the <body> tag (it should go before the {{body}} )